client/web/compose/src/components/ModuleFields/Configurator/File.vue

```diff
@@ -103,18 +95,13 @@
           md="6"
         >
           <b-form-group
-            :label="$t('kind.file.view.maxHeight')"
+            :label="$t('kind.file.view.maxHeight.label')"
+            label-class="text-primary"
           >
-            <b-input-group
-              :append="$t('kind.file.view.px')"
-            >
-              <b-form-input
-                v-model="f.options.maxHeight"
-                type="number"
-                number
-                :placeholder="$t('kind.file.view.maxHeight')"
-              />
-            </b-input-group>
+            <b-form-input
```

Comment:
Make sure you update the File block aswell, since they use same translations.
I would also not use placeholders here